### PR TITLE
run tests on server version only if content update was successfull

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/22348

### CODE REVIEW

File: Tests/scripts/is_content_installation_successful.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+from Tests.scripts.wait_until_server_ready import is_release_branch
+from Tests.test_utils import print_error, print_warning, print_color, LOG_COLORS
+from demisto_client.demisto_api.rest import ApiException
+from typing import List, AnyStr
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+def get_apikey_and_contentversion():
+    parser = argparse.ArgumentParser(description='Utility for batch action on incidents')
```
Comment (yuvalbenshalom) [inline Tests/scripts/is_content_installation_successful.py]: "Utility for batch action on incidents" is that the right description?

File: Tests/scripts/is_content_installation_successful.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            installed = resp_json.get("installed")
+            if not (release and content_version in release and notes and installed):
+                if is_release_branch():
+                    print_warning('On a release branch - ignoring content mismatch.')
+                else:
+                    print_error("Failed install content on instance [{}]\nfound content version [{}], expected [{}]"
+                                "".format(ami_instance_name, release, content_version))
+                    return False
+            else:
+                print_color("Instance [{instance_name}] content verified with version [{content_version}]".format(
```
Comment (yuvalbenshalom) [inline Tests/scripts/is_content_installation_successful.py]: Nice

**Comment by bakatzir** [comment]
Moved the check from a separate script in to the `update_content_on_demisto_instance` in `update_content_on_demisto_instance` script.

**Comment by bakatzir** [comment]
@yaakovi @yuvalbenshalom see the build i run with this change of code:
https://circleci.com/gh/demisto/content/43202
the relevant server versions failed with an indicative error.

**Comment by yuvalbenshalom** [comment]
@bakatzir looks like now it failed on the content update check.

**Comment by bakatzir** [comment]
@yuvalbenshalom Content version was not latest, so it failed which is good.
now I merged from master and it passes.

**Comment by bakatzir** [comment]
reference: https://github.com/demisto/etc/issues/22269

**Pull request merged**